src/Nordea/Components/Tooltip.elm

```diff
@@ -0,0 +1,161 @@
+        , content = []
+        }
+
+
+withPlacement : Placement -> Tooltip msg -> Tooltip msg
+withPlacement placement (Tooltip config) =
+    Tooltip { config | placement = placement }
+
+
+withAlignment : Alignment -> Tooltip msg -> Tooltip msg
+withAlignment alignment (Tooltip config) =
+    Tooltip { config | alignment = alignment }
+
+
+withContent : List (Html msg) -> Tooltip msg -> Tooltip msg
+withContent content (Tooltip config) =
+    Tooltip { config | content = content }
+
+
+view : List (Html msg) -> Tooltip msg -> Html msg
+view children (Tooltip config) =
+    styled span
+        [ Css.position Css.relative
+        , Css.hover
+            [ Css.descendants
+                [ Css.class "tooltip"
+                    [ Css.opacity (int 1) ]
+                ]
+            ]
+        ]
+        []
+        (children
+            ++ [ styled div
+                    [ Css.position Css.absolute
+                    , Css.backgroundColor Colors.grayDarkest
+                    , Css.color Colors.white
+                    , Css.padding2 (rem 0.5) (rem 1)
+                    , Css.borderRadius (rem 0.5)
+                    , Css.boxShadow4 zero (rem 0.0625) (rem 0.125) (Css.rgba 0 0 0 0.2)
+                    , Css.property "width" "max-content"
+                    , Css.maxWidth (rem 20)
+                    , Css.pointerEvents Css.none
+                    , Css.opacity zero
+                    , transition [ Css.Transitions.opacity 150 ]
+                    , tooltipContentStyle config.placement
+                    , Css.before
+                        [ Css.property "content" "' '"
+                        , Css.width (rem 0.625)
+                        , Css.height (rem 0.625)
+                        , Css.backgroundColor Css.inherit
+                        , Css.position Css.absolute
+                        , arrowStyle config.placement
+                        ]
+                    ]
+                    [ Attr.class "tooltip" ]
+                    config.content
+               ]
+        )
+
+
+tooltipContentStyle : Placement -> Style
+tooltipContentStyle placement =
+    case placement of
+        Top ->
+            Css.batch
+                [ Css.bottom (rem -1)
+                , Css.left (pct 50)
+                , Css.transform (Css.translate2 (pct -50) (pct 100))
+                ]
+
+        Bottom ->
+            Css.batch
+                [ Css.top (rem -1)
+                , Css.left (pct 50)
+                , Css.transform (Css.translate2 (pct -50) (pct -100))
+                ]
+
+        Left ->
+            Css.batch
+                [ Css.right (rem -1)
+                , Css.top (pct 50)
+                , Css.transform (Css.translate2 (pct 100) (pct -50))
+                ]
+
+        Right ->
+            Css.batch
+                [ Css.left (rem -1)
+                , Css.top (pct 50)
+                , Css.transform (Css.translate2 (pct -100) (pct -50))
+                ]
```

Comment:
I think we should change the meaning of the placement option so that `Top` means the tooltip is above the content and the arrow pointing down

Reply:
I actually agree, it's just that the Figma sketches are using placement this way. Have to change those as well 😄